Tools/pyco/pyco.py

```diff
@@ -270,7 +277,6 @@ def rewritten_bytecode(code: types.CodeType, builder: Builder) -> bytes:
                     raise RuntimeError(
                         f"More than 256 names in original {code.co_name} at line {code.co_firstlineno}"
                     )
-                oparg = builder.add_string(code.co_names[oparg])
                 if oparg >= 256:
```

Comment:
You could now implement EXTENDED_ARG properly (see line 237) and then you shouldn't need this test any more -- there would be no limit to how many names a code object can use.